modules/tinymce/src/core/main/ts/fmt/ExpandRange.ts

```diff
@@ -190,10 +186,12 @@ const findParentContainer = (
     if (!format[0].block_expand && dom.isBlock(parent)) {
       return parent;
     }
-
     // Walk left/right
     for (sibling = parent[siblingName]; sibling; sibling = sibling[siblingName]) {
-      if (!isBookmarkNode(sibling) && !isWhiteSpaceNode(sibling) && !isBogusBr(sibling)) {
+      const siblingIsNotSpace = NodeType.isText(sibling) && !onlyWhitespaces.test(sibling.data);
+      const isNotEmptyWithOnlySpaces = !(isEmpty(SugarElement.fromDom(sibling)) && siblingIsNotSpace);
```

Comment:
As discussed during the call last week, it'd probably be cleaner to do the not below instead. eg:

```
  const isEmptyWithOnlySpaces = isEmpty(SugarElement.fromDom(sibling)) && siblingIsNotSpace;

   if (!isBookmarkNode(sibling) && !isEmptyWithOnlySpaces && !isBogusBr(sibling)) {
```

Reply:
Sure. I removed all the Not, as to try and keep consistent between the two.